### DISCUSSION TITLE
Como medir produtividade?

### DISCUSSION THREAD

**Discussion by romulosardinha** [frontendbr/forum#1548] [Discussão]
Você trabalha home office ou no escritório? Freela ou não? Como medir sua produtividade com tantas variações de modos de trabalho por ai? 
Como vocês se avaliam?

**Comment by eliseumds** [comment]
Escritório.

Meço produtividade baseado em `([quanto querem me pagar] + [minha qualidade de vida] - [tarefas repetitivas]) / 3`. É extremamente difícil medir isso na nossa área.

Eu uso o TickTick para me organizar (Trello funciona também). Crio vários alertas no IFTTT e Google Calendar, evito tarefas repetitivas criando scripts que trabalhem pra mim. Tags na minha e-mail inbox. Zero notificações de Slack, FB, Twitter ou coisas assim durante horário de trabalho. Se algum app ou projeto open-source facilita meu trabalho, eu o compro ou faço uma doação.

**Comment by talitaazevedo** [comment]
Oie,
Eu uso bastante o trello,  utilizo o ifttt, para automatizar a criação de cartões como trabalho muito com e-mail, meus e-mails e tarefas do outlook acabam virando cards no trello.

Para não ficar perdendo tempo com rotina uso bastante o powershell para automatizar processos de máquina em servidores que utilizo por aqui.

**Comment by romulosardinha** [comment]
Trabalho com Freela Home office,
uso trello para organizar as etapas do projeto, google agenda para organizar o dia a dia.
Sempre tento medir e aperfeiçoar minha produtividade nos projetos.

Bater as metas do trello para cada item realizado no projeto e não deixar passar nenhuma das tarefas organizadas do google agenda é um bom medidor de produtividade para mim no momento.

Isso inclui estudos, meditação, academia, tempo de descanço... etc
Produtividade só tem sentido para mim se a qualidade de vida está em alta.
